src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -66,10 +66,6 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('collectors.xml');
```

Comment:
Ok, I see another solution, now that the `security.access.decision_manager` is properly decorated, just pass it to the collector instead of the `debug.security.access.decision_manager`, [this line](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php#L114) should ensure everything is fine, right?